content/en/docs/concepts/cluster-administration/api-priority-and-fairness.md

```diff
@@ -0,0 +1,125 @@
+---
+title: API priority and fairness
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
```

Comment:
What about a LIST across 10000 objects? Is that considered a long-running request as well?